### DISCUSSION TITLE
Minimal example of communication between AnyWidget and Zarr store to load data

### DISCUSSION THREAD

**Discussion by keller-mark** [manzt/anywidget#452] [Show and tell]
Based on the [Vizarr widget](https://github.com/hms-dbmi/vizarr/blob/ca1b1c5693f3cdc355a1e2f2f6b7bb57ba62d4ed/python/src/vizarr/_widget.py#L31) created by @manzt , this is an example of loading array data using Zarr (zarr-python on Python side and zarrita.js on the JS side): https://gist.github.com/keller-mark/b19c16434123b3b8cfbc05c8cd542f8e

**Comment by manzt** [comment]
Thanks for sharing! I've been wondering if we could move `send` into a small utility library.
